### PR TITLE
Participant state component

### PR DESCRIPTION
**Pull request by BroiSatse**
CPDRP-1015

Update the view participant details page shown to the school induction tutor to show eligibility results

### CODE REVIEW

File: app/components/schools/participants/status.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      end
+
+      def translation_scope
+        @translation_scope ||= "schools.participants.status.#{profile_status}"
+      end
+
+      def profile_status
+        if (eligibility = participant_profile.ecf_participant_eligibility)
+          return :ineligible if eligibility.ineligible_status?
+
```
Comment (ericaporter) [inline app/components/schools/participants/status.rb:29]: We needs to update the definition here for ineligibility, we aren't planning on relying on the ineligible status yet. Ineligibility for this release will be `ineligibility: { status: :manual_check, reason: %i[active_flags previous_induction] }) }`

Comment (ericaporter) [reply app/components/schools/participants/status.rb:29]: Well apparently we are now going to have ineligible, but I think we will still need to include the manual check / active_flags/previous_induction participants

**Review by kicferk1 — APPROVED** [review]

File: spec/components/schools/participants/status_spec.rb [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+            expect(rendered).to have_content I18n.t "schools.participants.status.eligible_fip.content"
+          end
+        end
+      end
+
+      context "when the participant has no QTS" do
+        let!(:eligibility) { create(:ecf_participant_eligibility, qts: false, participant_profile: profile) }
+
+        it "displays the fip ect no qts content" do
+          expect(rendered).to have_content I18n.t "schools.participants.status.fip_ect_no_qts.header"
```
Comment (ericaporter) [inline spec/components/schools/participants/status_spec.rb:63]: Nitpicking again "FIP no QTS"

Comment (ethanmills) [reply spec/components/schools/participants/status_spec.rb:63]: Ah, I left these lowercase to match the I18n key - the content is `:fip_ect_no_qts` : https://github.com/DFE-Digital/early-careers-framework/blob/CPDRP-1015-ParticipantDetails/config/locales/schools/participants.yml#L40

**Pull request closed without merging**